tests/ipsec/ipsec3hosts.pm

```diff
@@ -0,0 +1,126 @@
+# SUSE's openQA tests
+#
+# Copyright 2017-2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary:  Basic preparation before any IPSec test
+# Maintainer: Kernel QE <kernel-qa@suse.de>
+
+use Mojo::Base "ipsecbase", -signatures;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils;
+use lockapi;
+use network_utils;
+
+
+sub run ($self) {
+
+    select_serial_terminal;
+
+    $self->prepare_env();
+
+    # This case using following 3 hosts scenario
+    # host(left_ip) -------(middle_ip_01) router(middle_ip_01) ------ host(right_ip)
+
+    my $setup = {
+        'left_ip' => "2001:1:1:1::2",
+        'right_ip' => "2002:1:1:1::2",
+        'left_net' => "2001:1:1:1::/64",
+        'right_net' => "2002:1:1:1::/64",
+        'middle_ip_01' => "2001:1:1:1::1",
+        'middle_ip_02' => "2002:1:1:1::1",
+        'middle_net_01' => "2001:1:1:1::/64",
+        'middle_net_02' => "2002:1:1:1::/64",
```

Comment:
The original prefixes were different:
```
2001:0db8:ffff:0000::/64
2001:0db8:ffff:0001::/64
```
I hope using different addresses should be OK, but what was the reason to use different setup anyway?

Reply:
Just more short and simple character, no specific reason :)